i18n/en.json

```diff
@@ -1219,6 +1219,9 @@
   "admin.saml.firstnameAttrDesc": "(Optional) The attribute in the SAML Assertion that will be used to populate the first name of users in Mattermost.",
   "admin.saml.firstnameAttrEx": "E.g.: \"FirstName\"",
   "admin.saml.firstnameAttrTitle": "First Name Attribute:",
+  "admin.saml.guestAttrDesc": "(Optional) The attribute in the SAML Assertion that will be used to apply a guest role to users in Mattermost. If this attribute is removed/changed from your user in SAML and the user is still active, the user will be promoted to a member in Mattermost. Guests are prevented from accessing teams or channels upon logging in until they are assigned a team and at least one channel.",
```

Comment:
This PR probably isn't the best place for me to bring this up, but I hadn't considered it until I saw it written here... 

The "failure case" (of a filter being removed) grants *more* access to the users in question. Is this acceptable? I suppose everything would go back to normal if the filter were re-added so perhaps that makes it OK? Thoughts?

Reply:
@wiersgallak - @mkraft brings up an interesting point. If an administrator, for whatever reason, removes the guest filter, guests are granted additional access.  If they joined a team or channel before the filter was added back. I believe they would retain access to those teams/channels they joined while they had access. 
I wonder if perhaps a stronger warning if they change this field? Do we have that concept anywhere else, where we make them verify they want to make a risky change?

Reply:
@sbishel @mkraft Makes sense.  We do want this to be secure by default.  I propose because we have the ability through the System Console to promote a guest to a user, that we do not automate changing a guest to a user through this filter.  Chatted a bit with Scott, and we agree that if a user was previously in the user attribute and then shows up in the guest filter, they should automatically be demoted to a guest and the lowest permission set will be applied to the user.  

Would recommend setting text change to: 

"(Optional) The attribute in the SAML Assertion that will be used to apply a guest role to users in Mattermost. Guests are prevented from accessing teams or channels upon logging in until they are assigned a team and at least one channel. 

Note: If this attribute is removed/changed from your guest user in SAML and the user is still active, they will not be promoted to a member and will retain their Guest role. Guests can be promoted in **System Conole > User Mangement**.  Existing members that are identified by this attribute as a guest will be demoted from a member to a guest. ",